settings/settings.php

```diff
@@ -102,3 +109,26 @@ function user_busy_cost()
 	@define('CONST_Log_File_Format', 'TODO'); // Currently hard coded
 	@define('CONST_Log_File_SearchLog', '');
 	@define('CONST_Log_File_ReverseLog', '');
+
+	// handle http proxy when using file_get_contents
+	if (CONST_HTTP_Proxy) {
+		$proxy = 'tcp://' . CONST_HTTP_Proxy_Host . ':' . CONST_HTTP_Proxy_Port;
+		$aHeaders = array();
+		if(CONST_HTTP_Proxy_Login != null && CONST_HTTP_Proxy_Login != '' && CONST_HTTP_Proxy_Password != null && CONST_HTTP_Proxy_Password != '') {
+			$auth = base64_encode(CONST_HTTP_Proxy_Login . ':' . CONST_HTTP_Proxy_Password);
+			$aHeaders = array("Proxy-Authorization: Basic $auth", "Authorization: Basic $auth");
```

Comment:
Is the Authorization header correct here? I'd expect that Proxy-Authorization is sufficient.